pkg/scheduler/framework/job_updater.go

```diff
@@ -91,10 +91,11 @@ func isPodGroupStatusUpdated(newStatus, oldStatus scheduling.PodGroupStatus) boo
 func (ju *jobUpdater) updateJob(index int) {
 	job := ju.jobQueue[index]
 	ssn := ju.ssn
+	oldHyperNode := ssn.PodGroupOldState.Annotations[job.UID][api.TopologyAllocateLCAHyperNode]
 
 	job.PodGroup.Status = jobStatus(ssn, job)
-	oldStatus, found := ssn.podGroupStatus[job.UID]
-	updatePG := !found || isPodGroupStatusUpdated(job.PodGroup.Status, oldStatus)
+	oldStatus, found := ssn.PodGroupOldState.Status[job.UID]
```

Comment:
`oldHyperNode != job.PodGroup.GetAnnotations()[api.TopologyAllocateLCAHyperNode]` should be wrap as a func like `jobAllocatedHyperNodeChanged`.